weDevsOfficial/dokan
# chore: `DOKAN_SINCE` version replace command `npm run version` added

### All Submissions:






### Changes proposed in this Pull Request:


Version replace command `npm run version` or `grunt version` added.
This command will replace any occurrence of `DOKAN_SINCE` in the following files or folders,
```
    'assets/**/*',
    'deprecated/**/*',
    'includes/**/*',
    'templates/**/*',
    'src/**/*',
    'dokan.php',
    'uninstall.php',
```


### How to test the changes in this Pull Request:

1.  run `npm install`
2.  change any files listed above by adding `@since DOKAN_SINCE` in the comment or code.
3. run `npm run version` or `grunt version`
4. check the comment changed to `@since 3.4.0 // the current version number from package.json`



### Changelog entry

> No changelog entry needed


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
